pkg/background/generate/generate.go

```diff
@@ -341,7 +341,7 @@ func (c *GenerateController) applyGeneratePolicy(log logr.Logger, policyContext
 			return nil, processExisting, err
 		}
 
-		if !processExisting {
+		if policy.BackgroundProcessingEnabled() && rule.IsGenerateExisting() || !processExisting {
```

Comment:
1. Do we still need this flag `processExisting`?
2. Field `spec.background` controls whether the policy needs to be reconciled for policy reports. We should only rely on `generate.genExisting` to reconcile generate policies.

Reply:
Background check not required , missed to clean